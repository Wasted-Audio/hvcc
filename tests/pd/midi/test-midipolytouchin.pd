#N canvas 572 166 503 302 12;
#X obj 97 131 print;
#X obj 97 95 pack f f f;
#X obj 97 61 polytouchin;
#X connect 1 0 0 0;
#X connect 2 0 1 0;
#X connect 2 1 1 1;
#X connect 2 2 1 2;
