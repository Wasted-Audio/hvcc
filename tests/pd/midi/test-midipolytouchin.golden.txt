[@ 0.000] print: 0 22 1
[@ 0.000] print: 0 22 2
[@ 0.000] print: 1 22 1
[@ 0.000] print: 1 22 2
[@ 0.000] print: 2 22 1
[@ 0.000] print: 2 22 2
[@ 0.000] print: 3 22 1
[@ 0.000] print: 3 22 2
[@ 0.000] print: 4 22 1
[@ 0.000] print: 4 22 2
[@ 0.000] print: 5 22 1
[@ 0.000] print: 5 22 2
[@ 0.000] print: 6 22 1
[@ 0.000] print: 6 22 2
[@ 0.000] print: 7 22 1
[@ 0.000] print: 7 22 2
[@ 0.000] print: 8 22 1
[@ 0.000] print: 8 22 2
[@ 0.000] print: 9 22 2
[@ 0.000] print: 10 22 2
[@ 0.000] print: 11 22 2